app/assets/stylesheets/files.scss

```diff
@@ -94,6 +94,20 @@
   }
 }
 
+// style for popup error messages
+.white-popup {
+  position: relative;
+  background: #FFF;
+  padding: 20px;
+  width: auto;
+  max-width: 500px;
+  margin: 20px auto;
+}
+
+.mfp-iframe-holder .mfp-content {
+  max-width: 1200px;
```

Comment:
I suggest we use the variable `$max-width` instead of this. That way we can easily control how wide the system is allowed to be. Subtracting ~200 px from the max width leaves room for the gallery controls.

```sass
max-width: $max-width - 200px;
```